dist/pull.js

```diff
@@ -0,0 +1,50 @@
+"use strict";
```

Comment:
I think it will be great to follow the TypeScript naming scheme where source files are named as `src/**/*.ts` and the compiled code follows `lib/**/*.js` to keep editors happy. Most editors including VS Code and JetBrains will yell when they see both TypeScript and JavaScript are named with `js` extension.